code/src/components/NewThought.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+
+export const NewThought = ({ onChangeThought }) => {
+    const [newThought, setNewThought] = useState("")
+
+    const handleSubmit = e => {
+        e.preventDefault()
+        onChangeThought(newThought)
+        setNewThought('')
+    }
+
+    const handleMessages = e => {
+        setNewThought(e.target.value)
+    }
+
+    return (
+        <div className="new-thought-section">
+            <form onSubmit={handleSubmit}>
+                <label for="title-thought-section">What's making you happy right now?</label>
+                <textarea
+                    id="newHappyThought"
+                    name="newHappyThought"
+                    rows="5"
+                    cols="36"
+                    value={newThought}
+                    onChange={handleMessages}
+                    required
+                />
+            <br></br>
+            <button type="button" value="Submit">
```

Comment:
No functionality has been implemented to this button. In order for this button to do the submit for the form, you need to change the type to type="submit". That way the function handleSubmit gets called when the button is clicked 🙃

You are also missing the POST request in which we post a new thought to the API. This should probably be part of the onChangeThought function that is missing. The idea is that when you click on the button, we call the handleSubmit function which will then do a new fetch with method POST and add this new thought to the API 💌